src/views/Home/components/DaysLeft/index.jsx

```diff
@@ -1,11 +1,11 @@
 import { useEffect, useState } from 'react';
-import { Doughnut } from 'react-chartjs-2';
-import GordonLoader from 'components/Loader';
 import session from 'services/session';
 import styles from '../Doughnut.module.css';
-import { Card, CardContent, CardHeader, Grid, Typography } from '@mui/material';
+import { Card, CardContent, CardHeader, Grid, Typography, checkboxClasses } from '@mui/material';
```

Comment:
```suggestion
import { Card, CardContent, CardHeader, Grid, Typography } from '@mui/material';
```
I don't see where this import is being used